dashboard/client/src/service/log.ts

```diff
@@ -66,6 +66,10 @@ export type StateApiLogInput = {
    * If filename is provided, suffix is not necessary
    */
   filename?: string | null;
+  /**
+   * -1 for all lines.
```

Comment:
nit: my recollection of typescript typing is that the comment `-1 for all lines.` is an implementation detail and should be separate from the type declaration.

Reply:
I want this to be be part of the docs for this parameter so people who use this component know they can pass in -1